### PR TITLE
change try again to keep playing for some apps on freeplay

### PR DESCRIPTION
**Pull request by Bjvanminnen**

### CODE REVIEW

File: apps/src/templates/buttons.html.ejs [hunk 1]
```diff
@@ -30,7 +30,7 @@
   <% if (data.isK1 && !data.freePlay) { -%>
     <div id="again-button" class="launch arrow-container arrow-left">
       <div class="arrow-head"><img src="<%= data.assetUrl('media/tryagain-arrow-head.png') %>" alt="Arrowhead" width="67" height="130"/></div>
-      <div class="arrow-text"><%= msg.tryAgain() %></div>
+      <div class="arrow-text"><%= data.tryAgain %></div>
```
Comment (islemaster) [inline apps/src/templates/buttons.html.ejs]: Can `data.tryAgain` be `undefined` here?  Will it print "undefined"?

**Comment by islemaster** [comment]
Looks good, other than one question above.

File: apps/src/templates/buttons.html.ejs (hunk 1 above)
Comment (Bjvanminnen) [inline apps/src/templates/buttons.html.ejs]: It can't. We're inside a if (data.tryAgain) block

Comment (islemaster) [inline apps/src/templates/buttons.html.ejs]: :+1:

**Pull request merged**